File: backend/src/main/java/com/ll/hotel/domain/hotel/option/repository/HotelOptionRepository.java
package com.ll.hotel.domain.hotel.option.repository;

import com.ll.hotel.domain.hotel.option.entity.HotelOption;
import java.util.Collection;
import java.util.Set;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface HotelOptionRepository extends JpaRepository<HotelOption, Long> {
    Set<HotelOption> findByNameIn(Collection<String> names);
}

File: backend/src/main/java/com/ll/hotel/domain/hotel/option/repository/HotelOptionRepository.kt
package com.ll.hotel.domain.hotel.option.repository

import com.ll.hotel.domain.hotel.option.entity.HotelOption
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface HotelOptionRepository: JpaRepository<HotelOption, Long> {
    fun findByNameIn(names: Collection<String>): List<HotelOption>
}